src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-form big" data-qa="big">
+      <label class="search-lable search-lable-big">
+
+        <input
+        data-qa="keypress"
+        type="text"
+        class="input input-big"
+        placeholder="Try “Los Angeles”"
+        >
+      </label>
+    </form>
+
+
+
+    <form class="search-form small" data-qa="small">
+      <label class="search-lable search-lable-small">
+
+        <input
+        data-qa="hover"
+        type="text"
+        class="input input-small"
+        placeholder="Try “Los Angeles”"
+        >
```

Comment:
check this in all places
```suggestion
        <input
          data-qa="hover"
          type="text"
          class="input input-small"
          placeholder="Try “Los Angeles”"
        >
```